index.d.ts

```diff
@@ -1214,6 +1222,7 @@ declare namespace Eris {
       SYNC_CALL: 13;
     };
     ImageFormats: ["jpg", "jpeg", "png", "webp", "gif"];
+    GuildFeatures: ["ANIMATED_ICON" | "BANNER" | "COMMERCE" | "COMMUNITY" | "DISCOVERABLE" | "FEATURABLE" | "INVITE_SPLASH" | "MEMBER_VERIFICATION_GATE_ENABLED" | "NEWS" | "PARTNERED" | "PREVIEW_ENABLED" | "VANITY_URL" | "VERIFIED" | "VIP_REGIONS" | "WELCOME_SCREEN_ENABLED" | "TICKETED_EVENTS_ENABLED" | "MONETIZATION_ENABLED" | "MORE_STICKERS" | "THREE_DAY_THREAD_ARCHIVE" | "SEVEN_DAY_THREAD_ARCHIVE" | "PRIVATE_THREADS"];
```

Comment:
This means the array only contains 1 element which could be any of those values

Reply:
forgot to change the pipes to commas